ui/components/component-library/avatar-token/avatar-token.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { SIZES } from '../../../helpers/constants/design-system';
+
+import README from './README.mdx';
+import { AvatarToken } from './avatar-token';
+
+export default {
+  title: 'Components/ComponentLibrary/AvatarToken',
+  id: __filename,
+  component: AvatarToken,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    size: {
+      control: 'select',
+      options: Object.values(SIZES),
+    },
+    tokenName: {
+      control: 'text',
+    },
+    tokenImageUrl: {
+      control: 'text',
+    },
+    showHalo: {
+      control: 'boolean',
+    },
```

Comment:
I think we should add backgroundColor and borderColor here too. Will also need to add the color prop to the Box component so the fallback display color of the letter can change. I've created an issue here https://github.com/MetaMask/design-tokens/issues/213

Reply:
Done